src/state/models/feeds/posts.ts

```diff
@@ -397,6 +397,7 @@ export class PostsFeedModel {
       ]
     }
     if (this.feedType === 'home') {
+      // removing likedRepliesOnly will definitely show threads, but obviously it'll show unliked replies also
       return [FeedTuner.dedupReposts, FeedTuner.likedRepliesOnly]
```

Comment:
if you want to actually reproduce the screenshot I included in #746, you'll need to remove `likedRepliesOnly` from this